### ISSUE TITLE
Properly determine if a semantic node should be focusable

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#11179]
On iOS we currently use an inaccurate guess: https://github.com/flutter/engine/blob/master/shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm#L95
On Android we currently don't use setFocusable at all and leave it to the OS to guess what is focusable and what is not: https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo.html#setFocusable(boolean)
